src/lib/Scenes/Fair/Components/FairHeader/index.tsx

```diff
@@ -112,9 +176,15 @@ export class FairHeader extends React.Component<Props> {
         </BackgroundImage>
         <Spacer mt={2} />
         <Box mx={2}>{this.getContextualDetails()}</Box>
-        <Box px={2}>
+        <Box px={2} width={375} height={95}>
```

Comment:
Hey @ashleyjelks you have a set width here. This Box should be 100% width. I moved the one I added into a styled component to add `width: 100%;`

Reply:
Actually I wouldn't hardcode a height here either. The button component should know it's own size